app/models/supplier.rb

```diff
@@ -6,7 +6,7 @@ class Supplier
   attr_accessor :data
 
   delegate :name, :whitelabel_supplier, :slug,
-           :rank, :complaints_rating, :complaints_number,
+           :rank, :complaints_rating, :complaints_rating_score, :complaints_number,
```

Comment:
Might it be simpler to put the conditional logic here, so:

```rb
def complaints_rating
  if Feature.enabled?("FF_NEW_CSR_DATA")
    data.complaints_rating_score
  else
    data.complaints_rating
  end
end
```

Reply:
Ooooh, I like that!